presentation/src/main/java/me/androidbox/presentation/event/screen/EventScreenState.kt

```diff
@@ -0,0 +1,8 @@
+package me.androidbox.presentation.event.screen
+
+import android.net.Uri
+import androidx.compose.runtime.mutableStateListOf
+
+data class EventScreenState(
+    val listOfPhotoUri: List<Uri> = mutableStateListOf<Uri>()
```

Comment:
No need for mutableStateListOf<Uri>()
You could just:
= emptyList()

Reply:
@MKiperszmid I was thinking that adding to a mutablestatelistof will trigger this to re-compose

Reply:
No, it won't - you'll have to replace the whole list